### PR TITLE
added speakers!!

### PR DESCRIPTION
**Pull request by kendallwerts**
Please review and give feedback. I am also open to other things! Emily can add twitter handles later.

### CODE REVIEW

File: site/globals/schedule.json [hunk 1]
```diff
@@ -1,122 +1,80 @@
...
-  {
-    "start_time": "2:20pm",
+    "start_time": "1pm",
     "full": {
-      "title": "TALK_TITLE",
-      "speaker": {
-        "name": "SPEAKER_NAME",
-        "twitter": "twitter"
-      }
+      "title": "Lightning Talks"
```
Comment (maxxcrawford) [inline site/globals/schedule.json]: Should we include some sort of link for how to will work?

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Requested changes:

- [ ] Add more speaker info (Company name, Twitter @handle, rather than name being an invisible hyperlink) 
- [x] Remove the "What people are saying" section, or move it BELOW the speaker list
- [x] Optimize images (All JPG, optimized)

File: site/globals/schedule.json (hunk 1 above)
Comment (kendallwerts) [reply site/globals/schedule.json]: I don't think we know that yet.

**Comment by kendallwerts** [comment]
Working on Twitter handles and company names, but I'm putting the speakers up now.

**Pull request merged**